client/app/startup/OnboardingGuideApp.js

```diff
@@ -28,10 +28,10 @@ export default (props, railsContext) => {
     'edit_admin_listing_shape',
     'new_invitation',
     'new_listing',
-  ], { locale: railsContext.i18nLocale });
+  ], { locale: marketplaceContext.i18nLocale });
 
   const combinedReducer = combineReducers(reducers);
-  const combinedProps = Object.assign({}, props, { railsContext, routes });
```

Comment:
RailsContextReducer was removed (unused), so the store shouldn't get the context object anymore.